goinstant/prompts.go

```diff
@@ -537,6 +550,11 @@ func selectDefaultAction() error {
 		return nil
 	}
 
+	if result == "Help" {
```

Comment:
We should probably convert this to a switch statement now that there are at least 3 cases.